lib/cli-options.js

```diff
@@ -12,15 +15,80 @@ exports.CLIOptions = class {
     return parts.join('.');
   }
 
+  getProject() {
+    return JSON.parse(readFileSync([process.cwd(), 'aurelia_project', 'aurelia.json'].join('/')));
+  }
+
   getEnvironment() {
+    const project = this.getProject();
     let NODE_ENV;
-    let env = this.getFlagValue('env') || (process.env.NODE_ENV ? NODE_ENV = process.env.NODE_ENV : undefined) || 'dev';
+    let env = this.getFlagValue('env') || (process.env.NODE_ENV ? NODE_ENV = process.env.NODE_ENV : undefined) || project.build.defaultEnvironment || 'dev';
     if (NODE_ENV) {
       console.log(`The selected Node Environment (${NODE_ENV}) is not a preconfigured option ('dev', 'stage', and 'prod')`);
     }
     return env;
   }
 
+  getTarget(targetId, env) {
+    const project = this.getProject();
+    const targets = project.build.targets;
+    targetId = targetId || project.build.defaultTarget || 'web';
+    env = env || this.getEnvironment();
+
+    let target;
```

Comment:
Perhaps I'm missing something, but is this always going to return the first target if the `project.build.targets` is in fact an array?

Do you perhaps have an example of what the structure should look like?
My `aurelia.json` currently looks like this:
```
"build": {
    "targets": [
      {
        "id": "web",
        "displayName": "Web",
        "index": "index.html",
        "baseDir": ".",
        "output": "scripts"
      },
      {
        "id": "kiosk",
        "displayName": "Kiosk",
        "index": "kiosk.html",
        "baseDir": ".",
        "output": "kiosk_scripts"
      }
    ]
```